perfkitbenchmarker/virtual_machine.py

```diff
@@ -285,36 +314,65 @@ def RemoteCopy(self, file_path, remote_path='', copy_to=True):
                     (retcode, full_cmd, stdout, stderr))
       raise errors.VmUtil.SshConnectionError(error_text)
 
+  def _PushLongRunningCommandScripts(self):
+    """Pushes the scripts required by LongRunningRemoteCommand to this VM.
+
+    If the scripts have already been placed on the VM, this is a noop.
+    """
+    with self._has_remote_command_script_lock:
+      if not self._has_remote_command_script:
+        for f in (EXECUTE_COMMAND, WAIT_FOR_COMMAND):
+          self.PushDataFile(f, os.path.join(vm_util.VM_TMP_DIR,
+                                            os.path.basename(f)))
+
   def LongRunningRemoteCommand(self, command):
     """Runs a long running command on the VM in a robust way.
 
+    Executes a long running command via a pair of scripts on the VM:
```

Comment:
I removed "long running" from the docs here, and described the differences from RemoteCommand in more detail.